internal/db/schema/migrations/oss/postgres/56/05_mutable_ciphertext_columns.up.sql

```diff
@@ -32,4 +32,11 @@ begin;
   create trigger immutable_columns before update on worker_auth_authorized
     for each row execute function immutable_columns('worker_key_identifier', 'worker_id', 'worker_signing_pub_key', 'worker_encryption_pub_key', 'nonce', 'create_time');
 
+
+  -- With the addition of the registered tables test, we found one more table that needs its encrypted column made mutable
```

Comment:
😂. These comments are fun, but they are also a bit distracting tbh, maybe we should just remove them?